$OpenBSD: patch-src_command_c,v 1.8 2015/09/15 08:14:34 dcoppa Exp $

commit 2aba75803de7b4cc7319f804977b7a12a61b7f92
Author: Thorsten Wissmann <edu@thorsten-wissmann.de>
Date:   Tue Jul 28 12:09:24 2015 +0200

command: let listall print songs only, as described by the man page
already

--- src/command.c.orig	Fri Sep 11 14:23:47 2015
+++ src/command.c	Fri Sep 11 14:24:10 2015
@@ -541,7 +541,7 @@ cmd_listall(int argc, char **argv, struct mpd_connecti
 			if (!mpd_send_list_all_meta(conn, tmp))
 				printErrorAndExit(conn);
 
-			print_entity_list(conn, MPD_ENTITY_TYPE_UNKNOWN);
+			print_entity_list(conn, MPD_ENTITY_TYPE_SONG);
 		} else {
 			if (!mpd_send_list_all(conn, tmp))
 				printErrorAndExit(conn);
